lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
Yes, but if we agree that the method name is nicer in its shorter version (and it's definitely 'if' since it's a matter of taste), doesn't it make more sense to change later the one in metrics, instead of changing later both of them? (with the implication of fixing more places that call those methods)